assets/js/atomic/blocks/product-elements/stock-indicator/attributes.ts

```diff
@@ -1,4 +1,4 @@
-export const blockAttributes = {
+export const blockAttributes: Record< string, Record< string, unknown > > = {
 	productId: {
 		type: 'number',
 		default: 0,
```

Comment:
Thanks for the suggestion, @gigitux! Based on @sunyatasattva's comment in https://github.com/woocommerce/woocommerce-blocks/pull/7572#discussion_r1019471589, I've added an `interface` instead of a `type`.

Would you mind doing a final review?